mailbox/api/pom.xml

```diff
@@ -123,4 +123,17 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <reuseForks>true</reuseForks>
+                    <forkCount>8</forkCount>
```

Comment:
why 8? I would have expected 4 to be a better value (given that we are using 4 cores servers)

Reply:
Some of them are 8 (yoda, and another one).

Reply:
and you don't think we should take the lower value?

Reply:
Let's try with 4